enumerable.rb

```diff
@@ -0,0 +1,120 @@
+# rubocop :disable Style/CaseEquality
+
+# my_each
+module Enumerable
+  def my_each
+    return to_enumerable(:my_each) unless my_each_given?
```

Comment:
- [ ] on line #6 you use my_each_given? method but this method is not defined in your Enumerable module, let's define it or use a different way to make this validation work